<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KCFSelector</class>
 <widget class="QWidget" name="KCFSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KCF识别器</string>
  </property>
  <widget class="QLabel" name="show_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>641</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="select_btn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择ROI区域</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_video_btn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>290</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>加载视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="output_video_btn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>370</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>输出结果</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>KCF识别器</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camshift识别器</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>视频流</string>
   </property>
   <widget class="QRadioButton" name="camera_radbtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头视频流</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="file_radbtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>文件视频流</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
